extensions/agroal/deployment/src/main/java/io/quarkus/agroal/deployment/AgroalProcessor.java

```diff
@@ -20,11 +20,8 @@
 
 import io.agroal.api.AgroalDataSource;
 import io.quarkus.agroal.DataSource;
-import io.quarkus.agroal.runtime.AbstractDataSourceProducer;
-import io.quarkus.agroal.runtime.AgroalBuildTimeConfig;
-import io.quarkus.agroal.runtime.AgroalRuntimeConfig;
-import io.quarkus.agroal.runtime.AgroalTemplate;
-import io.quarkus.agroal.runtime.DataSourceBuildTimeConfig;
+import io.quarkus.agroal.runtime.*;
```

Comment:
We tend not to use import wildcards `*`. Can you revert the change?

Reply:
> We tend not to use import wildcards `*`. Can you revert the change?

done